pages/vi/profiles/JLKwong.md

```diff
@@ -0,0 +1,15 @@
+# **Name**: Jamie Kwong  
+# **Time Zone**: Pacific Standard Time  
+# **OS**: Windows 10, Version 10.0.19042, Build 19042
+
+I'm a curious and nerdy foodie who enjoys learning, reading, and having deep, esoteric conversations. In 2014, I experienced a catastrophic hardship, and there was pretty much no hope in me overcoming it - *at least, that’s what I thought*. Fast-forward to now, and I have track record above and beyond what I, or anyone else, believed I was capable of:
+
+* **Promoted early to a 25% increased pay rate** within 1 month by exceeding captioning metrics as a freelance captioning agent with Rev.  
+* **Obtained $10,000 in government funding** for people entering the workforce after a temporary disability by achieving a perfect score on the ACT Workkeys work performance assessment  
+* **Graduated 42 days early** by self-studying and passing the MTA: Networking Fundamentals certification exam ahead of schedule at the Professional Institute of Technology  
+* **Upgraded 80+ computers for 3 companies** from Windows 7 to Windows 10 using ConnectWise Control (ScreenConnect) and Windows Deployment Services at an IT internship with Brydan Solutions  
+* **Restored an empty 100-cubicle home office building back to full operations within 2 weeks** after a debilitating malware attack at Marshall Retail Group  
+* **Quarantined and restored 100+ PC's from scratch** after a devastating Mount Locker ransomware attack by identifying infected PC's, disconnecting them from the network, powering them off, imaging them, and reinstalling software according to each department's needs  
+* **Communicated hourly and daily updates with the IT team** during the Mount Locker ransomware attack to triage and restore the company home office back to full operations within 2 weeks
+
+I am a resilient, talented, and high-potential prospective software engineer intern at the Open Learning Exchange. By the end of 2021, I hope to kickstart my career in software engineering. :D Add me on LinkedIn: <a>https://www.linkedin.com/in/jamielk/</a>
```

Comment:
@JLKwong please use markdown and not html

Reply:
Oop, my bad. I'll fix that in a bit. xD

Reply:
I used a couple Markdown stuffs. It should be good now. ^-^